optaplanner-core/pom.xml

```diff
@@ -80,6 +80,10 @@
       <scope>test</scope>
     </dependency>
     <!-- XML -->
+    <dependency>
+      <groupId>com.thoughtworks.xstream</groupId>
+      <artifactId>xstream</artifactId>
+    </dependency>
```

Comment:
This should be either `<scope>test</scope>` or, even better, removed completely.